### PR TITLE
WX-1103 Accelerate TES CI

### PR DESCRIPTION
**Pull request by jgainerdewar**

### CODE REVIEW

**Comment by jgainerdewar** [comment]
This is ready for review, remaining test failures are being handled in other PRs.

File: supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala [hunk 1]
```diff
@@ -144,7 +144,6 @@ class TesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
         jobShell,
         mode)
     })
-
```
Comment (jgainerdewar) [inline supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala:147]: Will remove this inadvertent whitespace change shortly.

**Review by kraefrei — APPROVED** [review]
Looks good once the whitespace is cleared

File: src/ci/bin/testCentaurTes.sh [hunk 2]
```diff
@@ -60,7 +60,7 @@ startup_funnel
 
 # Limiting Centaur to 4 threads here in order to try to prevent exceeding the OS's max open files limit: BA-6153
```
Comment (THWiseman) [inline src/ci/bin/testCentaurTes.sh:61]: Do we want to change this comment to say `8` or remove it if it's no longer relevant?

**Comment by THWiseman** [comment]
INCREDIBLE gains holy cow! HUGE fan. 

Is there a comment you can leave near `TEST_THREAD_COUNT` that might clarify what it actually does (or doesn't) do and point future us towards the bash script instead?

**Review by THWiseman — APPROVED** [review]

**Pull request merged**